app/assets/javascripts/place.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
+# All this logic will automatically be available in application.js.
```

Comment:
Do we need this file? We're not using coffeescript in our projects

Reply:
sure - OK